ember-cli-build.js

```diff
@@ -3,7 +3,7 @@
 var path = require('path');
 var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 var EmberApp = require('ember-cli/lib/broccoli/ember-app');
-var shim = require('./lib/pod-templates-shim');
+var shim = require('flexi-layouts/lib/pod-templates-shim');
```

Comment:
Looks like I can't do the same in my consuming app. We'll have to figure out how to expose this shim from here.

`Could not require 'ember-cli-build.js': Cannot find module 'flexi-layouts/lib/pod-templates-shim'`